modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
 == Procedure
 To create a monitor:
 
-. In the *Functional Monitoring* section of Anypoint Monitoring, click *Create*.
-. Specify a name and a short description for the monitor.
-. Fill in the fields with the information that you have gathered.
-. Click *Create*.
-
-== Result
-
-The dashboard opens. The monitor that you created is listed on the left.
-
-Beneath the name of the monitor is an indicator that explains when the current run of the monitor began. A line at the bottom of the list item shows how much time remains before the next run of the monitor. Also, the *Monitored Endpoints*, *Tests*, and *Schedules* sections will all be empty.
-
-After the monitor has run, the list item for the test suite will indicate whether the monitored endpoints passed or failed.
+. Click *Create Monitor* in the *Functional Monitoring* section of Anypoint Monitoring.
+. Under *Step 1 - Setup the monitor*, specify a monitor name. Additionally, toggle *Validate SSL Certificate* to specify if the monitor needs to use self-signed certifications.
+. Tap the *Step 2 - Select endpoints* tab.
+. Populate the fields with the information gathered in "Before you Begin".
++
+You can preview the test by tapping *Run now* under *Preview the result of your monitor*.
```

Comment:
GLOBALLY search for "tap" and replace.
Up to this point, we have limited our UI verbs to "click" to initiate, "select" for menu or list choices, "check" and "uncheck" for checkboxes, and "toggle" for toggles.
Isn't "tap" a mobile-app verb?